packages/paste-core/components/badge/src/index.tsx

```diff
@@ -47,7 +47,6 @@ export const Badge = React.forwardRef<HTMLElement, BadgeProps>(
         alignItems="center"
         as={as}
         border="unset"
-        borderRadius="borderRadius30"
```

Comment:
Mistake?

Reply:
Purposeful. I need to test a change to prove it works